src/components/link/link.tsx

```diff
@@ -69,7 +69,8 @@ export interface LinkAnchorProps {
   type?: EuiLinkType;
   color?: EuiLinkColor;
   /**
-   * Set to true to show an icon indicating that it is an external link.
+   * Set to true to show an icon indicating that it is an external link. Defaults to true if
+   * `target="_blank".
```

Comment:
Nit picky formatting. Just makes it easier to read
```suggestion
   * Set to true to show an icon indicating that it is an external link;
   * Defaults to true if `target="_blank"`
```